### PR TITLE
Update error-codes.yaml

### PR DESCRIPTION
**Pull request by JakoobCZ**

### CODE REVIEW

**Comment by 3d-gussner** [comment]
@JakoobCZ Is this still an active PR? If so please fix the conflicts and also inform the Prusa-Firmware team about it as we need to update the message in MK3 branch.

File: 04_MMU/error-codes.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ Errors:
 
 - code: "04103"
   title: "FSENSOR DIDNT TRIGGER"
-  text: "Filament sensor didn't trigger while loading the filament. Ensure the filament reached the fsensor and the sensor works."
+  text: "Filament sensor didn't trigger while loading the filament. Ensure the sensor is calibrated and the filament reached it."
```
Comment (3d-gussner) [inline 04_MMU/error-codes.yaml:41]: This is useful for the MK4 but on the MK3/S/+ may be confusing.
`P`age
`L`ine on page
``` 
P1|01234567890123456789
L1|Filament sensor
L2|didn'trigger while
L3|loading the
L4|filament. Ensure the

P2|01234567890123456789
L1|sensor is calibrated
L2|and the filament
L3|reached it.
```
Alternative
```
P1|01234567890123456789
L1|Filament sensor
L2|didn'trigger while
L3|loading the
L4|filament. Ensure the

P2|01234567890123456789
L1|filament reached it
L2|and works correctly.
```

File: 04_MMU/error-codes.yaml [hunk 2]
```diff
@@ -83,7 +83,7 @@ Errors:
 
 - code: "04116"
   title: "SELECTOR CANNOT MOVE"
-  text: "The Selector cannot move. Check for anything blocking its movement. Check the wiring is correct."
+  text: "The Selector cannot move. Check for anything blocking its movement. Check if the wiring is correct."
```
Comment (3d-gussner) [inline 04_MMU/error-codes.yaml:86]:
```
P1|01234567890123456789
L1|The Selector cannot
L2|move. Check for
L3|anything blocking
L4|its movement. Check

P2|01234567890123456789
L1|if the wiring is
L2|correct.
```
Alternative 

```
P1|01234567890123456789
L1|The Selector cannot
L2|move. Check for
L3|anything blocking
L4|its movement and 

P2|01234567890123456789
L1|that the wiring is
L2|correct.
```

File: 04_MMU/error-codes.yaml [hunk 3]
```diff
@@ -97,7 +97,7 @@ Errors:
 
   code: "04126"
   title: "IDLER CANNOT MOVE"
-  text: "The Idler cannot move properly. Check for anything blocking its movement. Check the wiring is correct."
+  text: "The Idler cannot move properly. Check for anything blocking its movement. Check if the wiring is correct."
```
Comment (3d-gussner) [inline 04_MMU/error-codes.yaml:100]: Same as above

**Review by DRracer — APPROVED** [review]

**Pull request merged**